pupil_src/shared_modules/gaze_mapping/gazer_base.py

```diff
@@ -185,24 +197,53 @@ def fit_on_calib_data(self, calib_data):
         )
         # match pupil to reference data (left, right, and binocular)
         matches = self.match_pupil_to_ref(pupil_data, ref_data)
+        if matches.binocular[0]:
```

Comment:
@papr I'm a bit confused about why the check is `matches.binocular[0]` instead of `matches.binocular`. as far as I can see in `calibration_routines.data_processing.py: closest_matches_binocular_batch`, `matches.binocular/right/left` would return an empty list, so accessing the `0` index would raise `IndexError`.

please let me know if I'm missing something.

Reply:
Actually, the result of `closest_matches_binocular_batch()` goes into `matches.binocular`, not into `matches`. :)